CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKVisualProcessor.java

```diff
@@ -728,62 +742,36 @@ public int DeviceInitialised(CVKDevice cvkDevice) {
         
         // Scene knows about all renderable types so build the static descriptor layout
         // for each class.
-        assert(cvkDevice != null && cvkDevice.GetDevice() != null);
-        
-        // Initialise the shared atlas texture.  It extends renderable so it gets the call
-        // for updating shared resouces.  We could have a seperate render event listener but
-        // that seems like overkill for a single class.  For as long as it's a renderable it
-        // should be the first renderable so it gets updated before any of the objects that 
-        // depend on it.
-        cvkIconTextureAtlas = new CVKIconTextureAtlas(cvkDevice);
-        cvkRenderer.AddRenderable(cvkIconTextureAtlas);
+        CVKAssert(cvkDevice != null && cvkDevice.GetDevice() != null);
+              
         
         // Static as the shader and descriptor layout doesn't change per instance of renderable or over the course of the program
-        ret = CVKAxesRenderable.LoadShaders(cvkDevice);
+        ret = CVKAxesRenderable.StaticInitialise(cvkDevice);
```

Comment:
I think StaticInitialise should be called from the AddRenderable function (before Initialise()). As there will be other renderables added through different code paths we should have a generic way of doing this.

Reply:
We can't do that I don't think.  Static functions can't be inherited so you need the class name so it couldn't be in a function that takes a base class.  We could make each class responsible for calling their own static initialisers?

Reply:
I don't mind either way, but if you think it is a good idea can you add it in a change after this one please?

Reply:
Yep ok, I think when we add nodes we can revisit it and see if it should be changed.